test/test_fed.cpp

```diff
@@ -1193,25 +1193,25 @@ static void test_subtract(cindex_t dim, size_t size)
         }
         assert(fed3.eq(fed1));
         for (int count = 0; count < 500; ++count) {
-            if (test_generateRealPoint(pt, fed1))  // points in fed1 not in fed2
+            if (test_generatePoint(pt, fed1))  // points in fed1 not in fed2
```

Comment:
I do not like hiding fp issues under carpet, i.e. this fp test is important as it is, yet it's power is taken away by considering just integers.
If you insist on such a test (using the integers), then create a copy and leave this fp test alone for later.
I have a library which ignores the (lost) lower precision bits in the comparison, so we should use that instead.

Reply:
I don't think this is sweeping stuff under the carpet. It's more of an admission of the fact that you simply can't expect intersection to work properly when using floating point numbers, and therefore we shouldn't have a test that expects it. And then the `test_fp_intersection` test is a sort of documentation that shows why floating point numbers and intersections don't play well together. 

So basically, floating point clock valuation inside intersections are out of scope for the DBM project for now.